### DISCUSSION TITLE
passport-saml 4.0.5 release

### DISCUSSION THREAD

**Discussion by markstos** [node-saml/node-saml#320] [General]
@cjbarth I see passport-saml 4.0.5 was tagged 3 days ago, but I don't see it on NPM yet: https://www.npmjs.com/search?q=%40node-saml%2Fpassport-saml

Would you like me to do the publishing step?

**Comment by cjbarth** [comment]
It seems like it is published to me. `release-it` does that automatically; I didn't see errors, and this screenshot seems to indicate it worked fine:

![image](https://github.com/node-saml/node-saml/assets/3049726/df248b8b-a3be-4def-9b57-b8fe6ff8d110)

**Reply by markstos** [reply]
You are right. I was looking at passport-saml.  Should we also release a new version of `passport-saml` just to explicitly bump the minimum version of `node-saml` we depend on to increase the odds that users will get the most secure version?

**Reply by cjbarth** [reply]
They should be able to get an updated version automatically. Out `package.json` has a version that will pick this up. Once I sort out the `xml-crypto` stuff, I'll do a release that updates this to specify a minimum required version.

**Reply by markstos** [reply]
I was able to force the upgrade by adding this to my `package.json`, which forced Yarn to upgrade the node-saml sub-dependency:

```
  "resolutions": {
    "@node-saml/node-saml": "4.0.5"
  },
```
